File: Phase 2_PYTHON/break_continue.py
# break and continue keyword

i=1
while i<=10:
       if(i==5):
         print(i)
         break
       print(i)
       i+=1

print("\n")

i=1
while i<=10:
   if(i==5):
       i+=1
       continue
   print(i)
   i+=1

File: Phase 1 _JAVA/breakcontinue.java
public class breakcontinue 
{
    public static void main(String[] args) 
    {
         //continue keyword

        for(int i=1;i<=10;i++)
        {
            if(i==4)
            {
                continue;
            }
            System.out.println(i);
        }

        System.out.println();

        //break keyword
         for(int i=1;i<=10;i++)
        {
            if(i==4)
            {
                break;
            }
            System.out.println(i);
        }
     
    }

    
}
